versioned_docs/version-v0.10.0/int/quickstart/quickstart-group.md

```diff
@@ -176,7 +176,7 @@ A threshold of peers in the cluster need to perform this task to exit a validato
   - All VCs need to sign and submit the exact same messages (epoch) in DVT.
   - `--epoch=1` would be ideal, since all chains have that epoch in the past, so the validator should exit immediately.
   - There is however a [bug](https://github.com/sigp/lighthouse/issues/3471) in lighthouse requiring an epoch that maps to the latest fork version to be used.
-  - `compose-voluntary-exit.yml` is configured with `--epoch=112260` which is the latest Bellatrix fork on Prater.
+  - `compose-voluntary-exit.yml` is configured with `--epoch=162304` which is the latest Bellatrix fork on Prater.
```

Comment:
don't fix this, this is a versioned doc. Stuff needs to be fixed only in the `docs/` folder